docs/contracts.rst

```diff
@@ -862,6 +889,7 @@ Additional Resources for Understanding Events
 - `Javascript documentation <https://github.com/ethereum/wiki/wiki/JavaScript-API#contract-events>`_
 - `Example usage of events <https://github.com/debris/smart-exchange/blob/master/lib/contracts/SmartExchange.sol>`_
 - `How to access them in js <https://github.com/debris/smart-exchange/blob/master/lib/exchange_transactions.js>`_
+- `How Solidity Events Are Implemented (blog post) <https://blog.qtum.org/how-solidity-events-are-implemented-diving-into-the-ethereum-vm-part-6-30e07b3037b9>`_
```

Comment:
Do we want to link to a different blockchain?

Also do these sources actually explain anything beyond this documentation? If so, don't we want to include that detail?

Reply:
@axic It was talking about Ethereum, but sure, I get the point. I just thought it was really good. Maybe instead I'll remove it and add what's missing in our docs over time.